hs_core/models.py

```diff
@@ -3316,7 +3316,7 @@ def bag_checksum(self):
         """
         extra_data = self.extra_data
         if 'bag_checksum' in extra_data and extra_data['bag_checksum']:
-            return extra_data['bag_checksum']
+            return extra_data['bag_checksum'].strip('\n')
```

Comment:
What if we strip the checksum before saving it to extra_data instead?

Reply:
That is what we are doing now. This is just for handling potential extra ```\n``` in ```bag_checksum```  recorded in extra_data before the checksum strip() fix was put in. It is probably better to write a migration to handle this, but since strip() is simple and efficient, so I thought it is probably acceptable just to strip() it here again rather than writing a migration script to handle it. Let me know what you think is the better way.